src/Components/WorkloadsList/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import PageHeader from '@redhat-cloud-services/frontend-components/PageHeader';
+
+import messages from '../../Messages';
+import WorkloadsListTable from '../WorkloadsListTable';
+import { Title } from '@patternfly/react-core';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+
+const WorkloadsList = () => {
+  const intl = useIntl();
+  const chrome = useChrome();
+
+  chrome.updateDocumentTitle(
+    intl.formatMessage(messages.documentTitle, {
+      subnav: intl.formatMessage(messages.clusters),
+    })
+  );
+
+  return (
+    <React.Fragment>
+      <PageHeader className="ins-c-clusters-header">
+        <Title headingLevel="h1" ouiaId="page-header">
+          {`${intl.formatMessage(messages.insightsHeader)} ${intl
+            .formatMessage(messages.clusters)
+            .toLowerCase()}`}
```

Comment:
Let's start soft-deprecating `intl` and don't use it in new components. Feel free to change it to plain text

Reply:
Thanks, will do